Apps/Common/src/Delegates/ClientRegistriesDelegate.cs

```diff
@@ -194,10 +194,9 @@ private static bool SetNames(PN? nameSection, PatientModel patient)
 
         private static Address? MapAddress(AD? address)
         {
-            Address? retAddress = null;
-            if (address != null)
+            Address? retAddress = new ();
```

Comment:
There might be some side effects from changing this to return an empty Address instead of null.  I think the front-end swaps to the physical address when the postal address is null.